fem/restriction.cpp

```diff
@@ -725,7 +725,7 @@ H1FaceRestriction::H1FaceRestriction(const FiniteElementSpace &fes,
       MFEM_VERIFY(fe_dof_map.Size() > 0, "invalid dof map");
    }
    const TensorBasisElement* el =
-      dynamic_cast<const TensorBasisElement*>(fe);
+      dynamic_cast<const TensorBasisElement*>(fe0);
```

Comment:
Is `el` here the same as `tfe` above? @YohannDudouit

Reply:
Yes, but this code has also been completely refactored in #2195 , so I suspect that what these modifications are doing are mostly future merge conflicts.

Reply:
I agree, it would make sense to merge #2195 first, and then deal with the shadowing after that. I see that PR has two approvals already, is it waiting for @v-dobrev's final approval?

Reply:
I'm not a reviewer and I have not looked at this PR -- if you want me to look at something specific, just let me know.

Reply:
@v-dobrev, I was referring to #2195 (DG + PA + NC AMR). That PR has two approvals, I believe your approval is the last outstanding one.

Reply:
Oh, yes, I will finish my review of that PR soon.